src/containers/App/index.js

```diff
@@ -31,20 +37,37 @@ class App extends Component {
       <div className="appContainer">
         <HeaderContainer version={process.env.VERSION} />
         {this.props.hasWallet && <TransactionFloaterContainer />}
-        {this.props.children}
+        <TransitionGroup>
+          <CSSTransition key={currentKey} classNames="page-transition" timeout={timeout}>
+            {this.props.children}
+          </CSSTransition>
+        </TransitionGroup>
+        <Modal
+          isOpen={this.props.blockchainConnection && !this.props.account}
+          contentLabel="no-account-modal"
+          style={modalStyles}
+        >
+          <h1 id="heading">Oops!</h1>
+          <div id="description">
+            <p>We couldn't detect your account. Please check your wallet provider and reload the page</p>
```

Comment:
Can we fix this problem in Github where the syntax always messes up? That's btw why I always escape `'`